src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2284,41 +2284,47 @@ private IOperation CreateBoundDiscardPatternOperation(BoundDiscardPattern boundN
+                bool isImplicit = nameSyntax == syntax;
+                // We should expose the symbol in this case somehow:
+                // https://github.com/dotnet/roslyn/issues/33175
+                member = OperationFactory.CreateInvalidOperation(_semanticModel, nameSyntax, ImmutableArray<IOperation>.Empty, isImplicit);
+            }
+
             IPatternOperation pattern = (IPatternOperation)Create(subpattern.Pattern);
             return new PropertySubpatternOperation(member, pattern, _semanticModel, syntax, isImplicit: false);
         }
 
-        internal IOperation CreatePropertySubpatternMember(Symbol? symbol, ITypeSymbol matchedType, SyntaxNode syntax)
+        internal IOperation? CreatePropertySubpatternMember(BoundPropertySubpatternMember? member, ITypeSymbol matchedType)
         {
-            // PROTOTYPE(extended-property-patterns) ExpressionColon
-            var nameSyntax = (syntax is SubpatternSyntax subpatSyntax ? subpatSyntax.NameColon?.Name : null) ?? syntax;
-            bool isImplicit = nameSyntax == syntax;
-            switch (symbol)
+            switch (member?.Symbol)
             {
                 case FieldSymbol field:
                     {
                         var constantValue = field.GetConstantValue(ConstantFieldsInProgress.Empty, earlyDecodingWellKnownAttributes: false);
-                        return new FieldReferenceOperation(
-                            field.GetPublicSymbol(), isDeclaration: false, createReceiver(), _semanticModel, nameSyntax, field.Type.GetPublicSymbol(), constantValue, isImplicit: isImplicit);
+                        return new FieldReferenceOperation(field.GetPublicSymbol(), isDeclaration: false,
+                            createReceiver(), _semanticModel, member.Syntax, field.Type.StrippedType().GetPublicSymbol(), constantValue, isImplicit: false);
                     }
                 case PropertySymbol property:
                     {
-                        return new PropertyReferenceOperation(
-                            property.GetPublicSymbol(), ImmutableArray<IArgumentOperation>.Empty, createReceiver(), _semanticModel, nameSyntax, property.Type.GetPublicSymbol(),
-                            isImplicit: isImplicit);
+                        return new PropertyReferenceOperation(property.GetPublicSymbol(), ImmutableArray<IArgumentOperation>.Empty,
+                            createReceiver(), _semanticModel, member.Syntax, property.Type.StrippedType().GetPublicSymbol(), isImplicit: false);
                     }
                 default:
-                    // We should expose the symbol in this case somehow:
-                    // https://github.com/dotnet/roslyn/issues/33175
-                    return OperationFactory.CreateInvalidOperation(_semanticModel, nameSyntax, ImmutableArray<IOperation>.Empty, isImplicit);
+                    return null;
```

Comment:
>return null [](http://example.com/codeflow?start=20&length=11)

It looks like we're dropping the receiver in this case. Shouldn't we always call `createReceiver()` even if this member is invalid?